modules/krona/ktimporttaxonomy/main.nf

```diff
@@ -23,7 +23,10 @@ process KRONA_KTIMPORTTAXONOMY {
     script:
     def args = task.ext.args ?: ''
     """
-    ktImportTaxonomy "$report" -tax taxonomy
+    ktImportTaxonomy \\
+        "$report" \\
```

Comment:
I normaly rather put the input files at the end of teh command, but that's a minor quibble.